src/utils/update.ts

```diff
@@ -72,5 +72,5 @@ export function getNotifierMessage(updateInfo: UpdateNotifier.UpdateInfo) {
 }
 
 function getReleaseNotesUrl(majorVersion: number) {
-	return `https://github.com/balena-io/balena-cli/wiki/CLI-v${majorVersion}-Release-Notes`;
+	return `https://github.com/balena-io/balena-cli/releases/tag/v${majorVersion}.0.0`;
```

Comment:
Shouldn't it also take into account minor and patch?

Reply:
This notice only appears when the client is at least a major version behind. The point is to show what the major changes have been on the next major. So minor and patch are not intended to be shown here